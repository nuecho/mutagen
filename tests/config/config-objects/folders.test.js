const checkKey = (cfgObject) => {return cfgObject.name === "cfgObjectTest"};

cfgObjectTests("folders", "folders-config.json", {checkKey: checkKey, checkMandatoryProperties: false});
