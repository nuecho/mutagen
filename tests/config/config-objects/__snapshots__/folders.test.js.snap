// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[mutagen folders config import-export] should create the folders: the folders have been created 1`] = `
Object {
  "stderr": "",
  "stdout": "+ folder [type: 'person', owner: 'tenant/Environment', path: 'Persons/cfgObjectTest']
+ folder [type: 'person', owner: 'tenant/Environment', path: 'Persons/cfgObjectTest/personSubFolder']
Completed. 2 objects created. 0 object updated. 0 object skipped.
",
}
`;

exports[`[mutagen folders config import-export] should skip the same folders: the folders have not been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "= folder [type: 'person', owner: 'tenant/Environment', path: 'Persons/cfgObjectTest']
~ folder [type: 'person', owner: 'tenant/Environment', path: 'Persons/cfgObjectTest/personSubFolder']
Completed. 0 object created. 1 object updated. 1 object skipped.
",
}
`;

exports[`[mutagen folders config import-export] should update the modified folders: the modified folders have been updated 1`] = `
Object {
  "stderr": "",
  "stdout": "~ folder [type: 'person', owner: 'tenant/Environment', path: 'Persons/cfgObjectTest']
Completed. 0 object created. 1 object updated. 0 object skipped.
",
}
`;
